Universal Links vs Custom URL Schemes Support (#530)
italia/eidas-it-wallet-docs: docs/en/remote-flow.rst

   * - `client_id_schemes_supported`
     - RECOMMENDED. Array of Client Identifier schemes. Default is `entity_id`.
   * - `authorization_endpoint`

>      - URL of the authorization server's endpoint. See `OAUTH2`_, preferably using a universal link for enhanced security and fallback support, though a custom url-scheme can also be used if necessary.

Reviewer: ```suggestion
     - URL of the authorization server's endpoint, see `OAUTH2`_. Using an universal link is preferably for enhanced security and fallback support, custom url schemes can also be used if necessary.
```

  The ``wallet_nonce`` parameter is RECOMMENDED for Wallet Instances that wants to prevent reply of their http requests to the Relying Parties. 
  When present, the Relying Party MUST evaluate it.

.. note::

>   For the ``authorization_endpoint``, universal links are preferred over custom url-schemes due to several advantages: (1) When properly configured—using Assetlinks JSON for Android and Apple App Site Association for iOS—Universal Links provide enhanced security by reducing the risk of URL hijacking, 

Reviewer: ```suggestion
  For the ``authorization_endpoint`` the use of universal links are preferred over custom url-schemes because, when properly configured using Assetlinks JSON for Android and Apple App Site Association for iOS, they provide enhanced security by reducing the risk of URL hijacking. 
```

>   and (2) universal links offer a fallback mechanism, allowing the flow to continue seamlessly in a browser if the Wallet Instance is not installed, ensuring a smoother user experience. To ensure interoperability, support custom url-schemes is also provided by adhering to the OpenID4VC High Assurance Interoperability Profile (HAIP) recommendations `OPENID4VC-HAIP`_, including support for ``haip://``.

Reviewer: ```suggestion
  Furthermore, universal links offer fallback mechanisms, allowing the flow to continue seamlessly in a browser even if the Wallet Instance is not installed, ensuring a smoother User experience. To ensure interoperability, support custom url-schemes is also RECOMMENDED according to OpenID4VC High Assurance Interoperability Profile (HAIP)  `OPENID4VC-HAIP`_, and in particular using the custom url ``haip://``.
```

>   and (2) universal links offer a fallback mechanism, allowing the flow to continue seamlessly in a browser if the Wallet Instance is not installed, ensuring a smoother user experience. However, to ensure interoperability, we also support custom url-schemes by adhering to the OpenID4VC High Assurance Interoperability Profile (HAIP) recommendations `OPENID4VC-HAIP`_, including support for ``haip://``.

Reviewer: we do not use personal perspective using "we" "me" or "I" or "us" etc.

Author: done

> 
> 

Reviewer: ```suggestion
```
